packages/sdk/utils/src/account.ts

```diff
@@ -54,21 +55,20 @@ export async function generateMnemonic(
   return bip39ToUse.generateMnemonic(strength, undefined, getWordList(language))
 }
 
-export function validateMnemonic(
-  mnemonic: string,
-  defaultLanguage?: MnemonicLanguages,
```

Comment:
I'm not entire sure of removing these. In a way, if you know the language, you will avoid to search in every dictionary.
I'm more into the idea of searching in the dictionary that the user said, or in all of them otherwise (nil/null).

Thinking more of a common utils side, and not our use case in Valora

Reply:
I thought about my approach this way:

Good - eliminate the possibility of user error (i.e., if there is a mismatch between `defaultLanguage` and `mnemonic` language)

Bad - slightly less efficient as it defaults to checking all languages. However, there are not very many languages and `bip39.validateMnemonic` appears very optimized and will fail quickly on a language mismatch

What do you think?

Reply:
Ok, I agree. We could eventually have another function that requires an specific language if we see that we need to optimise that (which are not the actual cases due to we are only using these to create/update accounts, and won't be called too often)